gadgets/Makefile

```diff
@@ -99,12 +100,9 @@ test-ig-local: build
 	go test -exec 'sudo -E' -v ./...
 
 .PHONY:
-test-k8s: build
-	IG_PATH=$(KUBECTL_GADGET) \
-	IG_RUNTIME=kubernetes \
-	CGO_ENABLED=0 \
-	IG_EXPERIMENTAL=true \
-	GADGET_REPOSITORY=$(GADGET_REPOSITORY) \
-	GADGET_TAG=$(GADGET_TAG) \
-	IG_FLAGS=$(IG_FLAGS) \
-	go test -exec 'sudo -E' -v ./...
+test-local: IG_PATH=$(IG)
+test-local: test
```

Comment:
Probably because I never saw this - This confuses me having the same target under each other

Reply:
I didn't fin another way to do it, it's covered in https://stackoverflow.com/a/15230658 and https://www.gnu.org/software/make/manual/make.html#Target_002dspecific